src/dreambot/frontend/slack.py

```
@@ -1,5 +1,8 @@
 """Slack frontend for Dreambot."""
 import asyncio
+import base64
+import io
```

Ruff (0.8.2)

`io` imported but unused

Remove unused import: `io`

(F401)